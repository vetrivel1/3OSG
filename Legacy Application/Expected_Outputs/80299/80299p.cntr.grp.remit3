00050000546252848360,0000000001,08/25/2025,     576.50,11111,2180299,1000,2503,,,,48277150299
00050000546252848361,0000000002,08/25/2025,     709.48,11111,2180299,1000,2503,,,,48277150299
00050000546252848362,0000000003,08/25/2025,    1838.82,11111,2180299,1000,2503,,,,48277150299
00050000546252848363,0000000004,08/25/2025,     290.29,11111,2180299,1000,2503,,,,48277150299
00050000546252848364,0000000005,08/25/2025,    1245.46,11111,2180299,1000,2503,,,,48277150299
00050000546252848365,0000000006,08/25/2025,    1206.73,11111,2180299,1000,2503,,,,48277150299
00050000546252848366,0000000007,08/25/2025,    9729.53,11111,2180299,1000,2503,,,,48277150299
00050000546252848367,0000000008,08/25/2025,    5315.19,11111,2180299,1000,2503,,,,48277150299
00050000546252848368,0000000009,08/25/2025,    7334.80,11111,2180299,1000,2503,,,,48277150299
00050000546252848369,0000000010,08/25/2025,    1881.46,11111,2180299,1000,2503,,,,48277150299
00050000546252848370,0000000011,08/25/2025,    2918.02,11111,2180299,1000,2503,,,,48277150299
00050000546252848371,0000000012,08/25/2025,    3768.06,11111,2180299,1000,2503,,,,48277150299
00050000546252848372,0000000013,08/25/2025,   12947.28,11111,2180299,1000,2503,,,,48277150299
00050000546252848373,0000000014,08/25/2025,    4135.44,11111,2180299,1000,2503,,,,48277150299
00050000546252848374,0000000015,08/25/2025,     970.88,11111,2180299,1000,2503,,,,48277150299
00050000546252848375,0000000016,08/25/2025,    3318.24,11111,2180299,1000,2503,,,,48277150299
00050000546252848376,0000000017,08/25/2025,    2329.18,11111,2180299,1000,2503,,,,48277150299
00050000546252848377,0000000018,08/25/2025,    2793.52,11111,2180299,1000,2503,,,,48277150299
00050000546252848378,0000000019,08/25/2025,    4298.45,11111,2180299,1000,2503,,,,48277150299
00050000546252848379,0000000020,08/25/2025,    2294.52,11111,2180299,1000,2503,,,,48277150299
00050000546252848380,0000000021,08/25/2025,    3462.39,11111,2180299,1000,2503,,,,48277150299
00050000546252848381,0000000022,08/25/2025,     566.63,11111,2180299,1000,2503,,,,48277150299
00050000546252848382,0000000023,08/25/2025,    1539.07,11111,2180299,1000,2503,,,,48277150299
00050000546252848383,0000000024,08/25/2025,    3449.81,11111,2180299,1000,2503,,,,48277150299
00050000546252848384,0000000026,08/25/2025,    1574.56,11111,2180299,1000,2503,,,,48277150299
00050000546252848385,0000000028,08/25/2025,     408.91,11111,2180299,1000,2503,,,,48277150299
00050000546252848386,0000000029,08/25/2025,    3667.92,11111,2180299,1000,2503,,,,48277150299
00050000546252848387,0000000030,08/25/2025,    2760.74,11111,2180299,1000,2503,,,,48277150299
00050000546252848388,0000000031,08/25/2025,    2962.04,11111,2180299,1000,2503,,,,48277150299
00050000546252848389,0000000032,08/25/2025,    2149.85,11111,2180299,1000,2503,,,,48277150299
00050000546252848390,0000000033,08/25/2025,    1844.64,11111,2180299,1000,2503,,,,48277150299
00050000546252848391,0000000034,08/25/2025,     979.71,11111,2180299,1000,2503,,,,48277150299
00050000546252848392,0000000035,08/25/2025,     685.94,11111,2180299,1000,2503,,,,48277150299
00050000546252848393,0000000036,08/25/2025,     899.12,11111,2180299,1000,2503,,,,48277150299
00050000546252848394,0000000037,08/25/2025,    2455.63,11111,2180299,1000,2503,,,,48277150299
00050000546252848395,0000000038,08/25/2025,     133.79,11111,2180299,1000,2503,,,,48277150299
00050000546252848396,0000000039,08/25/2025,    2882.89,11111,2180299,1000,2503,,,,48277150299
00050000546252848397,0000000040,08/25/2025,    1804.65,11111,2180299,1000,2503,,,,48277150299
00050000546252848398,0000000041,08/25/2025,    1816.54,11111,2180299,1000,2503,,,,48277150299
00050000546252848399,0000000042,08/25/2025,    1384.65,11111,2180299,1000,2503,,,,48277150299
00050000546252848400,0000000043,08/25/2025,    1450.47,11111,2180299,1000,2503,,,,48277150299
00050000546252848401,0000000044,08/25/2025,     726.44,11111,2180299,1000,2503,,,,48277150299
00050000546252848402,0000000045,08/25/2025,    2516.24,11111,2180299,1000,2503,,,,48277150299
00050000546252848403,0000000046,08/25/2025,    4802.33,11111,2180299,1000,2503,,,,48277150299
00050000546252848404,0000000047,08/25/2025,    1152.88,11111,2180299,1000,2503,,,,48277150299
00050000546252848405,0000000048,08/25/2025,    5323.84,11111,2180299,1000,2503,,,,48277150299
00050000546252848406,0000000049,08/25/2025,    2742.70,11111,2180299,1000,2503,,,,48277150299
00050000546252848407,0000000050,08/25/2025,    2064.21,11111,2180299,1000,2503,,,,48277150299
00050000546252848408,0000000051,08/25/2025,    3772.82,11111,2180299,1000,2503,,,,48277150299
00050000546252848409,0000000052,08/25/2025,    2357.38,11111,2180299,1000,2503,,,,48277150299
00050000546252848410,0000000053,08/25/2025,    1800.69,11111,2180299,1000,2503,,,,48277150299
00050000546252848411,0000000054,08/25/2025,    2281.22,11111,2180299,1000,2503,,,,48277150299
00050000546252848412,0000000055,08/25/2025,    2086.24,11111,2180299,1000,2503,,,,48277150299
00050000546252848413,0000000056,08/25/2025,    1157.56,11111,2180299,1000,2503,,,,48277150299
00050000546252848414,0000000057,08/25/2025,    1745.67,11111,2180299,1000,2503,,,,48277150299
00050000546252848415,0000000058,08/25/2025,    3156.78,11111,2180299,1000,2503,,,,48277150299
00050000546252848416,0000000059,08/25/2025,    4921.02,11111,2180299,1000,2503,,,,48277150299
