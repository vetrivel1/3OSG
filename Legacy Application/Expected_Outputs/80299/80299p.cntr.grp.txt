000001|0000000001          |0002|000| |Y| | | | | | | | | | |N|807         |807         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|1
000001|0000000001          |0002|000| |Y| | | | | | | | | | |N|807         |807         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|0
000002|0000000002          |0002|000| |Y| | | | | | | | | | |N|SR1         |SR1         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|1
000002|0000000002          |0002|000| |Y| | | | | | | | | | |N|SR1         |SR1         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|0
000003|0000000003          |0002|000| |Y| | | | | | | | | | |N|492         |492         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|1
000003|0000000003          |0002|000| |Y| | | | | | | | | | |N|492         |492         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|0
000004|0000000004          |0002|000| |Y| | | | | | | | | | |N|139         |139         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|1
000004|0000000004          |0002|000| |Y| | | | | | | | | | |N|139         |139         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|0
000005|0000000005          |0002|000| |Y| | | | | | | | | | |N|492         |492         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|1
000005|0000000005          |0002|000| |Y| | | | | | | | | | |N|492         |492         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|0
000006|0000000006          |0002|000| |Y| | | | | | | | | | |N|815         |815         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|1
000006|0000000006          |0002|000| |Y| | | | | | | | | | |N|815         |815         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|0
000007|0000000007          |0002|000| |Y| | | | | | | | | | |N|178         |178         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|1
000007|0000000007          |0002|000| |Y| | | | | | | | | | |N|178         |178         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|0
000008|0000000008          |0002|000| |Y| | | | | | | | | | |N|492         |492         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|1
000008|0000000008          |0002|000| |Y| | | | | | | | | | |N|492         |492         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|0
000009|0000000009          |0002|000| |Y| | | | | | | | | | |N|104         |104         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|1
000009|0000000009          |0002|000| |Y| | | | | | | | | | |N|104         |104         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|0
000010|0000000010          |0002|000| |Y| | | | | | | | | | |N|104         |104         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|1
000010|0000000010          |0002|000| |Y| | | | | | | | | | |N|104         |104         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|0
000011|0000000011          |0002|000| |Y| | | | | | | | | | |N|304         |304         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|1
000011|0000000011          |0002|000| |Y| | | | | | | | | | |N|304         |304         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|0
000012|0000000012          |0002|000| |Y| | | | | | | | | | |N|165         |165         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|1
000012|0000000012          |0002|000| |Y| | | | | | | | | | |N|165         |165         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|0
000013|0000000013          |0002|000| |Y| | | | | | | | | | |N|492         |492         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|1
000013|0000000013          |0002|000| |Y| | | | | | | | | | |N|492         |492         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|0
000014|0000000014          |0002|000| |Y| | | | | | | | | | |N|492         |492         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|1
000014|0000000014          |0002|000| |Y| | | | | | | | | | |N|492         |492         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|0
000015|0000000015          |0002|000| |Y| | | | | | | | | | |N|480         |480         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|1
000015|0000000015          |0002|000| |Y| | | | | | | | | | |N|480         |480         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|0
000016|0000000016          |0002|000| |Y| | | | | | | | | | |N|492         |492         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|1
000016|0000000016          |0002|000| |Y| | | | | | | | | | |N|492         |492         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|0
000017|0000000017          |0002|000| |Y| | | | | | | | | | |N|178         |178         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|1
000017|0000000017          |0002|000| |Y| | | | | | | | | | |N|178         |178         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|0
000018|0000000018          |0002|000| |Y| | | | | | | | | | |N|178         |178         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|1
000018|0000000018          |0002|000| |Y| | | | | | | | | | |N|178         |178         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|0
000019|0000000019          |0002|000| |Y| | | | | | | | | | |N|178         |178         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|1
000019|0000000019          |0002|000| |Y| | | | | | | | | | |N|178         |178         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|0
000020|0000000020          |0002|000| |Y| | | | | | | | | | |N|326         |326         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|1
000020|0000000020          |0002|000| |Y| | | | | | | | | | |N|326         |326         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|0
000021|0000000021          |0002|000| |Y| | | | | | | | | | |N|104         |104         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|1
000021|0000000021          |0002|000| |Y| | | | | | | | | | |N|104         |104         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|0
000022|0000000022          |0002|000| |Y| | | | | | | | | | |N|807         |807         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|1
000022|0000000022          |0002|000| |Y| | | | | | | | | | |N|807         |807         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|0
000023|0000000023          |0002|000| |Y| | | | | | | | | | |N|304         |304         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|1
000023|0000000023          |0002|000| |Y| | | | | | | | | | |N|304         |304         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|0
000024|0000000024          |0002|000| |Y| | | | | | | | | | |N|492         |492         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|1
000024|0000000024          |0002|000| |Y| | | | | | | | | | |N|492         |492         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|0
000025|0000000026          |0002|000| |Y| | | | | | | | | | |N|394         |394         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|1
000025|0000000026          |0002|000| |Y| | | | | | | | | | |N|394         |394         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|0
000026|0000000028          |0002|000| |Y| | | | | | | | | | |N|816         |816         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|1
000026|0000000028          |0002|000| |Y| | | | | | | | | | |N|816         |816         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|0
000027|0000000029          |0002|000| |Y| | | | | | | | | | |N|492         |492         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|1
000027|0000000029          |0002|000| |Y| | | | | | | | | | |N|492         |492         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|0
000028|0000000030          |0002|000| |Y| | | | | | | | | | |N|334         |334         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|1
000028|0000000030          |0002|000| |Y| | | | | | | | | | |N|334         |334         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|0
000029|0000000031          |0002|000| |Y| | | | | | | | | | |N|818         |818         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|1
000029|0000000031          |0002|000| |Y| | | | | | | | | | |N|818         |818         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|0
000030|0000000032          |0002|000| |Y| | | | | | | | | | |N|104         |104         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|1
000030|0000000032          |0002|000| |Y| | | | | | | | | | |N|104         |104         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|0
000031|0000000033          |0002|000| |Y| | | | | | | | | | |N|834         |834         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|1
000031|0000000033          |0002|000| |Y| | | | | | | | | | |N|834         |834         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|0
000032|0000000034          |0002|000| |Y| | | | | | | | | | |N|816         |816         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|1
000032|0000000034          |0002|000| |Y| | | | | | | | | | |N|816         |816         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|0
000033|0000000035          |0002|000| |Y| | | | | | | | | | |N|144         |144         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|1
000033|0000000035          |0002|000| |Y| | | | | | | | | | |N|144         |144         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|0
000034|0000000036          |0002|000| |Y| | | | | | | | | | |N|807         |807         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|1
000034|0000000036          |0002|000| |Y| | | | | | | | | | |N|807         |807         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|0
000035|0000000037          |0002|000| |Y| | | | | | | | | | |N|443         |443         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|1
000035|0000000037          |0002|000| |Y| | | | | | | | | | |N|443         |443         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|0
000036|0000000038          |0002|000| |Y| | | | | | | | | | |N|834         |834         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|1
000036|0000000038          |0002|000| |Y| | | | | | | | | | |N|834         |834         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|0
000037|0000000039          |0002|000| |Y| | | | | | | | | | |N|816         |816         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|1
000037|0000000039          |0002|000| |Y| | | | | | | | | | |N|816         |816         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|0
000038|0000000040          |0002|000| |Y| | | | | | | | | | |N|178         |178         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|1
000038|0000000040          |0002|000| |Y| | | | | | | | | | |N|178         |178         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|0
000039|0000000041          |0002|000| |Y| | | | | | | | | | |N|818         |818         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|1
000039|0000000041          |0002|000| |Y| | | | | | | | | | |N|818         |818         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|0
000040|0000000042          |0002|000| |Y| | | | | | | | | | |N|F01         |F01         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|1
000040|0000000042          |0002|000| |Y| | | | | | | | | | |N|F01         |F01         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|0
000041|0000000043          |0002|000| |Y| | | | | | | | | | |N|834         |834         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|1
000041|0000000043          |0002|000| |Y| | | | | | | | | | |N|834         |834         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|0
000042|0000000044          |0002|000| |Y| | | | | | | | | | |N|104         |104         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|1
000042|0000000044          |0002|000| |Y| | | | | | | | | | |N|104         |104         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|0
000043|0000000045          |0002|000| |Y| | | | | | | | | | |N|308         |308         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|1
000043|0000000045          |0002|000| |Y| | | | | | | | | | |N|308         |308         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|0
000044|0000000046          |0002|000| |Y| | | | | | | | | | |N|373         |373         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|1
000044|0000000046          |0002|000| |Y| | | | | | | | | | |N|373         |373         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|0
000045|0000000047          |0002|000| |Y| | | | | | | | | | |N|834         |834         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|1
000045|0000000047          |0002|000| |Y| | | | | | | | | | |N|834         |834         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|0
000046|0000000048          |0002|000| |Y| | | | | | | | | | |N|443         |443         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|1
000046|0000000048          |0002|000| |Y| | | | | | | | | | |N|443         |443         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|0
000047|0000000049          |0002|000| |Y| | | | | | | | | | |N|178         |178         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|1
000047|0000000049          |0002|000| |Y| | | | | | | | | | |N|178         |178         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|0
000048|0000000050          |0002|000| |Y| | | | | | | | | | |N|306         |306         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|1
000048|0000000050          |0002|000| |Y| | | | | | | | | | |N|306         |306         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|0
000049|0000000051          |0002|000| |Y| | | | | | | | | | |N|104         |104         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|1
000049|0000000051          |0002|000| |Y| | | | | | | | | | |N|104         |104         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|0
000050|0000000052          |0002|000| |Y| | | | | | | | | | |N|104         |104         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|1
000050|0000000052          |0002|000| |Y| | | | | | | | | | |N|104         |104         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|0
000051|0000000053          |0002|000| |Y| | | | | | | | | | |N|178         |178         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|1
000051|0000000053          |0002|000| |Y| | | | | | | | | | |N|178         |178         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|0
000052|0000000054          |0002|000| |Y| | | | | | | | | | |N|104         |104         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|1
000052|0000000054          |0002|000| |Y| | | | | | | | | | |N|104         |104         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|0
000053|0000000055          |0002|000| |Y| | | | | | | | | | |N|834         |834         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|1
000053|0000000055          |0002|000| |Y| | | | | | | | | | |N|834         |834         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|0
000054|0000000056          |0002|000| |Y| | | | | | | | | | |N|SR1         |SR1         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|1
000054|0000000056          |0002|000| |Y| | | | | | | | | | |N|SR1         |SR1         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|0
000055|0000000057          |0002|000| |Y| | | | | | | | | | |N|SR1         |SR1         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|1
000055|0000000057          |0002|000| |Y| | | | | | | | | | |N|SR1         |SR1         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|0
000056|0000000058          |0002|000| |Y| | | | | | | | | | |N|815         |815         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|1
000056|0000000058          |0002|000| |Y| | | | | | | | | | |N|815         |815         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|0
000057|0000000059          |0002|000| |Y| | | | | | | | | | |N|585         |585         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|1
000057|0000000059          |0002|000| |Y| | | | | | | | | | |N|585         |585         |            |                         |                                        |                                                  |80299p.cntr.grp               |00001|00002|0
